include/cantera/base/ct_defs.h

```diff
@@ -49,62 +49,83 @@ using std::isnan; // workaround for bug in libstdc++ 4.8
  */
 
 //! Pi
-const doublereal Pi = 3.14159265358979323846;
+const double Pi = 3.14159265358979323846;
 
 /*!
- *   @name Variations of the Gas Constant
- * %Cantera uses the MKS system of units. The unit for moles
- * is defined to be the kmol.
+ * @name Defined Constants
+ * These constants are defined by CODATA to have a particular value.
+ * https://physics.nist.gov/cgi-bin/cuu/Category?view=html&Adopted+values.x=71&Adopted+values.y=14
```

Comment:
I agree the link is not ideal. I wanted to document that these numbers are the ones defined specifically under the "Defined Constants" page, i.e., they have no uncertainty because they are not measured. Do you think that distinction is worth preserving?

Reply:
I think noting that these are defined constants with exact values is useful. I just don't think the deep link is needed. Anyone who wants to check on these values can go to the higher-level page and search for the specific constant of interest.